common/src/main/java/org/apache/sedona/common/raster/RasterOutputs.java

```diff
@@ -101,4 +104,11 @@ public static byte[] asArcGrid(GridCoverage2D raster, int sourceBand) {
     public static byte[] asArcGrid(GridCoverage2D raster) {
         return asArcGrid(raster, -1);
     }
+
+    public static String asBase64(GridCoverage2D raster) throws IOException {
```

Comment:
Please add a test in Sedona-common to test it.

Reply:
BTW, I am adding tests for the other outputs as well. Don't merge rn.

Reply:
arc grid test is very slow, I didn't include it.